src/assets/css/sections/benefits.css

```diff
@@ -0,0 +1,19 @@
+.benefits-section {
+    position: relative;
+    top: -218px;
+    left: 7.4%;
+}
+
+.benefits-section .benefits-header {
+    color: #000;
```

Comment:
create variable for color

Reply:
:(( I was created but I forgot use it, thanks